frontend/app/module/NotFound.tsx

```diff
@@ -2,24 +2,21 @@ import { Error, ErrorAction } from "app/component/error";
 import { Footer } from "app/component/footer/Footer";
 import { NavBar } from "app/component/navbar/NavBar";
 
+import { t, TranslationPath, tx } from "@kiesraad/i18n";
 import { AppLayout } from "@kiesraad/ui";
 
 export interface NotFoundProps {
-  message?: string;
-  path?: string;
+  message: TranslationPath;
+  path: string;
 }
 
 export function NotFound({ message, path }: NotFoundProps) {
```

Comment:
`PageNotFound` noes not always cover what is happening - an URL / path might be valid and found, but a part of the fetched data might be missing. I will postfix both the fatal error and the not found page with the word `Page`, since they both render a full page message.